feed-ingress/cmd/common_test.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/sky-uk/feed/nginx"
+)
+
+func TestCreatePortsConfigWithoutPorts(t *testing.T) {
+	if os.Getenv("BE_CRASHER") == "1" {
+		createPortsConfig(unset, unset)
+		return
+	}
+	cmd := exec.Command(os.Args[0], "-test.run=TestCreatePortsConfigWithoutPorts")
+	cmd.Env = append(os.Environ(), "BE_CRASHER=1")
+	err := cmd.Run()
+	if e, ok := err.(*exec.ExitError); ok && !e.Success() {
+		return
+	}
+	t.Fatalf("process ran with err %v, want exit status 1", err)
```

Comment:
just a minor thing but we could say 'process exited with %v, want exit status 1", err' as I think this could be exit status 0